src/snowflake/snowpark/modin/plugin/_internal/generator_utils.py

```diff
@@ -223,12 +223,8 @@ def generate_irregular_range(
         while start + periods * offset <= end:
             periods += 1
 
-    # TODO: SNOW-1646883 fix invalid identifier error when sql_simplifier_enabled is True
     session = get_active_session()
-    sql_simplifier_enabled = session.sql_simplifier_enabled
```

Comment:
@sfc-gh-aalam is this fixed https://snowflakecomputing.atlassian.net/browse/SNOW-1646883, what is the problem ?

Reply:
I don't know. It used to raise error for test described in the ticket. Seems to have fixed now so this change is no longer required

Reply:
also @sfc-gh-nkrishna that code seems again violates the architecture in our Snowpark Pandas, that there should be no direct interact with snowpark dataframe, the right way is to provide a static function inside ordered dataframe for range here, and return an ordered dataframe. Then all rest operations should happen on the ordered dataframe. can we get a refactoring task to refact that part?